task/pins/index.js

```diff
@@ -0,0 +1,64 @@
+const fs = require('fs')
+const util = require('util')
+const path = require('path')
+
+const { parseIcon, combinePictoPin, getColoredPin } = require('../../lib/svg_icon');
+
+const excludeList = [
+  // transports
+  'aerialway',
+  'airport',
+  'bus',
+  'entrance',
+  'ferry',
+  'parking',
+  'parking-bike',
+  'rail',
+  'rail-light',
+  'rail-metro',
+  // street furniture
+  'barrier',
+  'information',
+  'post-box',
+  'telephone',
+  'waste-basket',
+  // place types
+  'home',
+  'residential-community',
+  'town',
+];
+
+module.exports = async (options) => {
+  const readFile = util.promisify(fs.readFile)
+  const readdir = util.promisify(fs.readdir)
+
+  const iconFiles = await readdir(`${path.resolve(options.styleDir)}/icons`)
+  const pinStream = await readFile('../../pin-mini.svg');
+  fs.mkdirSync(path.join(options.styleDir, 'build/pins'), { recursive: true });
+
+  const pinPromises = iconFiles
+    /* use hi-res versions only */
```

Comment:
you are using the low-res version after all

Reply:
Oups, the comment is outdated. I reverted to low-res as they are the ones used for the current background POI layers and I wanted the pins to be consistent with them.
The hi-res are the ones used to generate the icon font.
Let's decide later if we can keep only one version.